test/kitchen/test/integration/dd-agent-runtime-security-test/rspec/dd-agent-runtime-security-test_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'spec_helper'
+
+if os != :windows
```

Comment:
Nit: The `if` is not needed since we don't run the test on Windows anyway.

Reply:
Removed the `if`